apps/test/unit/p5lab/spritelab/spriteCommandsTest.js

```diff
@@ -140,11 +140,11 @@ describe('Sprite Commands', () => {
       ).to.equal(10);
     });
 
-    it('caps at 500 sprites', () => {
+    it('caps at 1000 sprites', () => {
```

Comment:
There's some great functionality added here. To ensure it keeps working in the long-term, add a few more tests to cover added scenarios. For example:
* display only the last 1000 lines of text
* calling makeNumSprites 3 times with 500 sprites should still cap at 1000 sprites